clusterloader2/pkg/test/teststruct.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package test
+
+import (
+	"fmt"
+
+	"k8s.io/perf-tests/clusterloader2/pkg/framework"
+	"k8s.io/perf-tests/clusterloader2/pkg/state"
+)
+
+type testStruct struct {
+	framework *framework.Framework
+	state     *state.NamespacesState
+	done      <-chan struct{}
+	namespace string
+	prefix    string
+}
+
+func (ts *testStruct) setNamespace(ns string) *testStruct {
```

Comment:
I think you can remove this set functions. They're just called at one place.

Reply:
function removed.